experimental/trusted_shuffler/backend/src/main.rs

```diff
@@ -36,12 +39,27 @@ pub struct Opt {
     listen_address: String,
 }
 
+lazy_static! {
+    // We use START_TIME to time the time elapsed between every request and response.
```

Comment:
Is it though? AFAICT only request times are logged? Is the idea that callers can figure out the elapsed time by comparing the time on the logs?

Reply:
But why not get a new instant every time instead of trying to synchronize on a global instant?